src/decayangle/decay_topology.py

```diff
@@ -418,6 +418,7 @@ def helicity_angles(
         return HelicityAngles(
             -minus_phi_rf,
             -minus_theta_rf,
+            self.mass(momenta),
         )
```

Comment:
Oh, the method is called "HelicityAngles", that already no does not reflect trues with possibility to use other conventions

Maybe it is a chance to change it

Reply:
something towards `KinematicVariables`